lib/src/library_analyzer.dart

```diff
@@ -54,8 +54,24 @@ class LibraryScanner {
     var pubPackageMapProvider = new PubPackageMapProvider(
         PhysicalResourceProvider.INSTANCE, sdk, runPubList);
     var packageMapInfo = pubPackageMapProvider.computePackageMap(
-        PhysicalResourceProvider.INSTANCE.getResource(projectPath) as Folder);
+        PhysicalResourceProvider.INSTANCE.getResource(packagePath) as Folder);
     var packageMap = packageMapInfo.packageMap;
+
+    var packageNames = <String>[];
+    packageMap.forEach((k, v) {
+      if (v.any((f) => p.isWithin(packagePath, f.path))) {
```

Comment:
This to prevent a wild import of '../../../../../etc/passwd'? Maybe we should flag it in the package analysis / general health score?